src/applications/coronavirus-research/config/form.js

```diff
@@ -83,18 +85,24 @@ export function transform(formConfig, form) {
 }
 
 const formConfig = {
+  rootUrl: manifest.rootUrl,
   urlPrefix: '/',
   submitUrl: `${environment.API_URL}/covid-research/volunteer/create`,
   trackingPrefix: 'covid-research-volunteer-',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
   transformForSubmit: transform,
   formId: VA_FORM_IDS.FORM_COVID_VACCINE_TRIAL,
+  benefitDescription: {
+    benefitType: 'coronavirus volunteer list',
+    subText: false,
+  },
   version: 0,
   prefillEnabled: true,
   customText: {
     reviewPageTitle: 'Review information',
     appType: 'volunteer form',
```

Comment:
**Note:** This will make the form saved message read, "Your coronavirus volunteer list volunteer form has been saved."